confdgnmi/src/confd_gnmi_api_adapter.py

```diff
@@ -1,11 +1,15 @@
+import os
+import select
 import sys
+import threading
 import xml.etree.ElementTree as ET
-from socket import socket
+from socket import socket, AF_INET, SOCK_STREAM
 
 import _confd
 # TODO replace maapi_low with high level MAAPI
```

Comment:
I know this is not part of this PR, but still: you are not actually using it at all (for anything useful), you can just remove it; the same holds for maapisock.

Reply:
will remove `maapisock` as class attribute

Reply:
unused imports removed